src/index.ts

```diff
@@ -118,26 +202,29 @@ Upload.prototype.createURI = function (callback) {
     reqOpts.headers!.Origin = this.origin;
   }
 
-  this.makeRequest(reqOpts, function (err, resp) {
-    if (err) return callback(err);
+  this.makeRequest(reqOpts, (err: Error, resp: r.RequestResponse) => {
+    if (err) {
+      return callback(err);
+    }
 
     const uri = resp.headers.location;
-    self.uri = uri;
-    self.set({ uri });
-    self.offset = 0;
+    this.uri = uri;
+    this.set({ uri });
+    this.offset = 0;
 
     callback(null, uri);
   });
 };
 
 Upload.prototype.continueUploading = function () {
-  if (typeof this.offset === 'number') return this.startUploading();
+  if (typeof this.offset === 'number') {
+    return this.startUploading();
+  }
   this.getAndSetOffset(this.startUploading.bind(this));
 };
 
-Upload.prototype.startUploading = function () {
+Upload.prototype.startUploading = function (){
```

Comment:
We went `() {` => `(){` here, but not in `continueUploading`. Is there a linter or autoformatter that we can use to keep everything consistent?

Reply:
Good question!  This is what `gts fix` should be doing.  Right now it's failing because we have *a lot* of lint errors that I haven't wanted to fix just yet.  I can go through and do that next 👍

Reply:
Ah, gotcha!